beacon-chain/sync/subscriber.go

```diff
@@ -765,3 +766,10 @@ func multiAddr(pid peer.ID, stat *peers.Status) string {
 	}
 	return addrs.String()
 }
+
+func rejectGossipMessage(msg *pubsub.Message) pubsub.ValidationResult {
+	if features.Get().EnableFullSSZDataLogging {
+		log.WithField("rawMessage", msg).Debug("Rejected gossip message with data")
```

Comment:
If you want to match Teku's log, they also log the reason or error.

https://github.com/ConsenSys/teku/blob/cdf161742a7abb091ab04fb6c23fee622b9d1df7/infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/P2PLogger.java#L50